### ISSUE TITLE
Provide tui-editor file containing all dependencies and extensions

### ISSUE THREAD

**Issue by Gilles-GitHub** [nhn/tui.editor#556]
Hello,

Do you provide a file containing tui-editor + its dependencies (jquery, ...) + the tui-editor extensions ?

I have an Angular app and I succeeded using tui-editor in it.
In order to make it work I need to download the _tui-editor-Editor-full.js_ file from the CDN and put it in a folder and load it in my Angular 7 app using:
```javascript
import Editor from 'src/assets/js/tui-editor-Editor-full';
```
Now I need to use the table extension but when I try to import it the same way as I imported the _tui-editor-Editor-full_ script:
```javascript
import 'src/assets/js/tui-editor-extTable';
```
I got these errors:
```
WARNING in ./src/assets/js/tui-editor-extTable.js
Module not found: Error: Can't resolve 'tui-editor' in 'C:\workspace\src\assets\js'

WARNING in ./src/assets/js/tui-editor-extTable.js
Module not found: Error: Can't resolve 'tui-editor/dist/tui-editor-Viewer' in 'C:\workspace\src\assets\js'

ERROR in ./src/assets/js/tui-editor-extTable.js
Module not found: Error: Can't resolve 'jquery' in 'C:\workspace\src\assets\js'
ERROR in ./src/assets/js/tui-editor-extTable.js
Module not found: Error: Can't resolve 'to-mark' in 'C:\workspace\src\assets\js'
ERROR in ./src/assets/js/tui-editor-extTable.js
Module not found: Error: Can't resolve 'tui-code-snippet' in 'C:\workspace\src\assets\js'
i ｢wdm｣: Failed to compile.
```

The easiest solution for me would be to have a single js file containing the editor, it's dependencies and it's extensions.
Would you provide something like that ?

I understand that it's a particular case because tui-editor was not made for Angular and I should use a package manager (unfortunately I can't due to some restrictions on my side) but after 3 days on it I did not find a solution.
I even post the pb on stackoverflow hoping for some help from Angular experts here is the [link](https://stackoverflow.com/questions/56602815/module-not-found-error-cant-resolve-when-using-external-js-library)

**Comment by Gilles-GitHub** [comment]
As it's a special request and not something that could be usefull for many people I made my own _tui-editor-Editor-full.js_ file using your repository and modifying the webpack config.
I close the issue.

**Comment by sohee-lee7** [comment]
`tui-editor-Editor-full.js` should have all dependencies and extensions. So this issue will be fixed v1.4.4.
